### DISCUSSION TITLE
How to query RealmDictionary fields in kotlin ?

### DISCUSSION THREAD

**Discussion by Mr4rogrammer** [realm/realm-kotlin#1678] [General]
Any idea

**Comment by rorbech** [comment]
Realm Kotlin supports querying on dictionary field through [Realm Query Language](https://www.mongodb.com/docs/realm/realm-query-language/). You can see some examples of queries on object by dictionary predicates [here](https://www.mongodb.com/docs/realm/realm-query-language/#dictionary-operators). 

As an example, this would return all `Sample`-objects where the `intDictionaryField` has the key `key1`. 
```
realm.query<Sample>("ANY intDictionaryField.@keys == 'key1'").find()
```
